Laser/styles/general.css

```diff
@@ -0,0 +1,84 @@
+body {
+    width: 1024px;
+    margin: 0 auto;
+    margin-top: 220px;
+    font-family: 'Playfair Display', serif;
+}
+
+header {
+    background: var(--header-bg-color);
+    width: 1024px;
+    position: fixed;
+    top: 0;
+    z-index: 10;
+}
+
+.cap p {
+    font-size: 1.5em;
+    line-height: 10px;
+    font-weight: bold;
+}
+
+.cap, nav, .services {
+    display: flex;
+    justify-content: space-between;
+}
+
+a::before, h1::before{
+    content: '';
+    display: inline-block;
+    height: 100%;
+    vertical-align: middle;
+}
+
+.logo {
+    margin-top: 12px;
+    width: 490px;
+    height: 90px;
+}
+
+h1, a {
+    background: var(--main-bg-color);
+    color: var(--main-text-color);
+    width: 100%;
+    height: 55px;
+    border-radius: 10px;
+    margin: 2px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    font-size: 2em;
+    font-weight: 500;
+    text-align: center;
+    text-decoration: none;
+}
+
+h1 {
+    background: var(--h1-bg-color);
+    color: var(--h1-text-color);
+}
+
+a:hover {
+    background: var(--main-hover-color);
+}
+
+.services {
+        /* Для візуалізаціїї скрола */
+        margin-bottom: 900px;
+}
+
+:root {
+    --main-text-color: #F5EDF0;
+    --main-bg-color: #424C55; 
+    --main-hover-color: #8895A0;
+    --h1-bg-color: #424C55;
+    --main-bg-color: white;
+}
+
+:root {
+    --main-text-color: #aac6d0;
+    --main-bg-color: #04548c;
+    --main-hover-color: #2281ee;
+    --h1-bg-color: black;
+    --h1-text-color: black;
+    --header-bg-color: black;
+}
+
```

Comment:
you had to create the same variable for different themes, but you created only for root
so home task is not completely done